libr/io/io_fd.c

```diff
@@ -126,3 +126,42 @@ R_API int r_io_fd_get_current(RIO *io) {
 	}
 	return -1;
 }
+
+R_API int r_io_fd_get_next(RIO *io, int fd) {
```

Comment:
r_io_get_next_fd maybe?

Reply:
no, it's r_<big_api>_<small_api>_<function_name>